src/main/java/org/folio/consortia/domain/entity/UserTenantEntity.java

```diff
@@ -0,0 +1,54 @@
+package org.folio.consortia.domain.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.Objects;
+import java.util.UUID;
+
+@Getter
+@Setter
+@ToString
```

Comment:
You can use just lombok "Data" annotanion and remove Getter, Setter, ToString, RequiredArgsConstructor as well as equals and hashcode

Reply:
@Data is giving warn message 
![image](https://user-images.githubusercontent.com/113523904/224667492-4b6c2329-406b-47ac-bfa4-15a3de823cf4.png)
https://www.jpa-buddy.com/blog/lombok-and-jpa-what-may-go-wrong/